crypto/fipsmodule/sha/asm/keccak1600-armv8.pl

```diff
@@ -458,6 +458,8 @@
 	mov	$out,x1
 	mov	$len,x2
 	mov	$bsz,x3
+	cmp	x4, #0				// x4 = 'next' argument
```

Comment:
```suggestion
	cmp	x4, #0		// x4 = 'padded' argument; if !=0, perform Keccak first
```

Reply:
fixed in [3240910](https://github.com/aws/aws-lc/pull/1839/commits/32409105f83366884c41183fdd46d75fd57f2367)